ocs_ci/utility/openshift_console.py

```diff
@@ -0,0 +1,112 @@
+import os
+import logging
+from datetime import datetime
+
+from ocs_ci.framework import config
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.exceptions import (
+    CommandFailed,
+    OpenshiftConsoleSuiteNotDefined,
+    UnsupportedBrowser,
+)
+from ocs_ci.ocs.ocp import OCP
+from ocs_ci.utility.utils import get_kubeadmin_password, run_cmd
+
+logger = logging.getLogger(__name__)
+
+
+class OpenshiftConsole:
+
+    def __init__(self, browser=constants.CHROME_BROWSER):
+        self.browser = browser
+        self.console_path = config.RUN['openshift_console_path']
```

Comment:
I think init can setup the clone and set the path and we dont have to depend on user to do this.